pwa/src/components/applications/applicationForm.tsx

```diff
@@ -147,6 +148,13 @@ export const ApplicationForm: React.FC<ApplicationFormProps> = ({id}) => {
                         <Spinner/>
                       ) : (
                         <div>
+                          {saveSpinner === true ? (
+                            <div className="spinner-overlay">
+                              <div className="spinner-overlay-content">
+                                <Spinner />
+                              </div>
+                            </div>
+                          ) : ( <div></div>)}
```

Comment:
Instead of using the syntax:

```
{ saveSpinner === true ? <></> : <></> }
```

You can use:

```
{ saveSpinner && <></> }
```

This has a couple of benefits:

1. Less code
2. Better readability
3. You are no longer rendering an empty element (the `<div></div>`) when the `saveSpinner` state is false.

Also, for future reference, when you're checking if a boolean is true, you do not have to explicitly say so. Resulting in: 

```
if (saveSpinner === true)
if (saveSpinner)
```